src/vsr/journal.zig

```diff
@@ -1837,24 +1855,31 @@ pub fn Journal(comptime Replica: type, comptime Storage: type) type {
         }
 
         pub fn writing(self: *Self, op: u64, checksum: u128) bool {
+            const slot = self.slot_for_op(op);
             var it = self.writes.iterate();
+            var found: bool = false;
```

Comment:
```suggestion
            var found: bool = false;
            var it = self.writes.iterate();
```

To keep the `it` variable closer to where it's used in the loop below, and also since `found` is the function's return value and more significant.